### PR TITLE
Riya parquet

### PR DESCRIPTION
**Pull request by riyamhatre**
- incorporated important code from master and verified that it works with parquet
- run the "load data" code that is commented in pipeline to generate the parquet files

### CODE REVIEW

File: src/risknet/config/conf.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+general:
```
Comment (rodalbuyeh) [inline src/risknet/config/conf.yaml:1]: why is this coming in as a new thing in the git diff? didn't we already have this?

File: src/risknet/proc/label_prep.py [hunk 2]
```diff
@@ -7,10 +7,16 @@
 from typing import List, Dict, Tuple
 import pickle
 import logging
+#import dask.dataframe as dd #use dask in place of pandas
+logger = logging.getLogger("freelunch")
```
Comment (rodalbuyeh) [inline src/risknet/proc/label_prep.py:11]: logger initialization should occur after all imports.

File: src/risknet/proc/parquet.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                                       "insurance_recoveries", "net_sales_proceeds", "non_insurance_recoveries",
+                                       "expenses",
+                                       "legal_costs", "maintenance_costs", "taxes_and_insurance", "misc_expenses",
+                                       "actual_loss", "modification_cost", "step_modification_flag",
+                                       "deferred_payment_modification", "loan_to_value", "zero_balance_removal_upb",
+                                       "delinquent_accrued_interest","del_disaster","borrower_assistance","month_mod_cost","interest_bearing"]
+
+    monthly['row_hash'] = monthly.assign(partition_count=50).partition_count.cumsum() % 50
+
+    monthly.to_parquet("src/risknet/data/" + '/monthly.parquet', partition_on = "row_hash")
```
Comment (rodalbuyeh) [inline src/risknet/proc/parquet.py:31]: lgtm

File: src/risknet/proc/parquet.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    org = dd.read_csv(fm_root+data2, sep='|', header = None,dtype={25: 'object',
+           26: 'object',
+           28: 'object'})
+    org = org.drop(columns = {26, 27,28,29,30,31})
+    org.columns = ["credit_score", "first_payment_date", "first_time_homebuyer", "maturity_date",
+                                           "metropolitan_division", "mortgage_insurance_percent", "number_of_units",
+                                           "occupancy_status", "orig_combined_loan_to_value", "dti_ratio",
+                                           "original_unpaid_principal_balance", "original_ltv", "original_interest_rate",
```
Comment (rodalbuyeh) [inline src/risknet/proc/parquet.py:40]: you might consider moving this stuff to a config somewhere

File: src/risknet/run/pipeline.py [hunk 5]
```diff
@@ -112,7 +112,4 @@
 
 print(auc)
 print(pr)
-print(recall)
-
-
-    
+print(recall)
```
Comment (rodalbuyeh) [inline src/risknet/run/pipeline.py:115]: log, don't print

**Review by rodalbuyeh — APPROVED** [review]

File: src/risknet/config/conf.yaml (hunk 1 above)
Comment (riyamhatre) [reply src/risknet/config/conf.yaml:1]: Oh I realized that I had named it something else on github so I renamed the file

**Review by zaki1503 — APPROVED** [review]

**Pull request merged**